CHANGELOG.md

```diff
@@ -5,14 +5,42 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-
 ## [Unreleased]
- 
+
 ### Added
+
 ### Changed
+
+- **BREAKING CHANGE** [Modals](nulogy.design/components/Modal/) no longer accept `primaryButton` or `secondaryButtons` props and instead have `footerContent` that can accept a React node, allowing more flexibility in the content of a Modal's footer.
+
+  Old API:
+
+  ```
+  <Modal
+
```

Comment:
missing primaryButton, and secondaryButtons props in the code sample here?

Reply:
lol wtf where did it go... will add back now...